spec/features/shelters/3user_can_see_one_shelter_spec.rb

```diff
@@ -1,35 +1,55 @@
 require 'rails_helper'
 
-RSpec.describe "When I visit a shelter's show page,", type: :feature do
-  it "I see a list of reviews for that shelter" do
-    shelter_1 = Shelter.create(name: "Dumb Friends League",
-                              address: "1191 Yuma St",
-                              city: "Denver",
-                              state: "CO",
-                              zip: 80223)
-    shelter_2 = Shelter.create(name: "Pick a Pet",
-                              address: "123 Fake St",
-                              city: "Denver",
-                              state: "CO",
-                              zip: 80209)
-
-    visit "/shelters/#{shelter_1.id}"
-
-    expect(page).to have_content(shelter_1.name)
-    expect(page).to have_content("Address: #{shelter_1.address}")
-    expect(page).to have_content("City: #{shelter_1.city}")
-    expect(page).to have_content("State: #{shelter_1.state}")
-    expect(page).to have_content("Zip: #{shelter_1.zip}")
-    expect(page).not_to have_content(shelter_2.name)
+RSpec.describe "Shleter page", type: :feature do
```

Comment:
"Youll see in the first commit a chunk of code got deleted (copy and pasted wrong thing) it looks scary but I put it all back in in a later commit."

This is what i was talking about but got added back in